src/ucp/core/ucp_context.c

```diff
@@ -878,6 +910,20 @@ ucs_status_t ucp_config_modify(ucp_config_t *config, const char *name,
         return status;
     }
 
+    if (ucs_global_opts_is_unmodifiable(name)) {
```

Comment:
we can probably remove "ucs_global_opts_set_value_modifiable" function now, and use only 'ucs_global_opts_set_value' - after checking ucs_global_opts_is_readonly